### PR TITLE
[vpa] update readme

### PR DESCRIPTION
**Pull request by mcook-bison**
**Why This PR?**
_a short description of why this PR is needed_
Updates documentation to reduce confusion.

Fixes #

**Changes**
Changes proposed in this pull request:

* minor correction to example command in readme.

**Checklist:**

* [x] I have updated the chart version in `Chart.yaml` following Semantic Versioning.
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] Any new values have been added to the README for the Chart, or helm-docs has been run for the charts that support it.

### CODE REVIEW

**Review by sudermanjr — APPROVED** [review]
Thanks!!!!

File: stable/vpa/README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ This chart is entirely based on the manifests and various scripts in the `deploy
 
 ## Tests and Debugging
 
-There are a few tests included with this chart that can help debug why your installation of VPA isn't working as expected. You can run `helm -n <Release Namespace> <Release Name>` to run them.
+There are a few tests included with this chart that can help debug why your installation of VPA isn't working as expected. You can run `helm test -n <Release Namespace> <Release Name>` to run them.
```
Comment (sudermanjr) [inline stable/vpa/README.md:9]: Premature approval, sorry.

You'll want to add this to Readme.md.gotmpl and then use https://github.com/norwoodj/helm-docs to regenerate the README. Let me know if you need anything for that.

**Review by sudermanjr — CHANGES_REQUESTED** [review]

**Comment by mcook-bison** [comment]
Sorry!  Will correct.
________________________________

File: stable/vpa/README.md (hunk 1 above)
Comment (mcook-bison) [reply stable/vpa/README.md:9]: Sorry about that.  I think it's corrected now.  After running helm-docs I do see a difference below ('Chart Values' to 'Values') that doesn't seem to be caused by me... perhaps caused by a different version of helm-docs?

Comment (sudermanjr) [reply stable/vpa/README.md:9]: I'll get this merged https://github.com/FairwindsOps/charts/pull/371 and that should take care of it.

**Review by sudermanjr — CHANGES_REQUESTED** [review]
Really annoying, but you're right, helm-docs changed the format. I merged the PR and updated your branch, but you'll have to bump the chart version again for that. Thanks!

**Pull request merged**